src/setup/msw/handlers.ts

```diff
@@ -0,0 +1,11 @@
+import { http, HttpResponse } from 'msw'
+import { fakeDb } from '@todo/utils';
+import { API_URL } from '@todo/consts';
+
+const handlers = [
+  http.get(`${API_URL}/items`, async () => {
+    return await HttpResponse.json(fakeDb.todoList, { status: 200 });
```

Comment:
I believe that you don't need to add `{ status: 200 }`, it's by default for all successful responses in msw.